coursera_autograder/commands/upload.py

```diff
@@ -263,16 +263,21 @@ def command_upload(args):
 
     # Rebuild an authorizer to ensure it's fresh and not expired
     auth = oauth2_instance.build_authorizer()
-    grader_id = register_grader(auth,
-                                args,
-                                bucket=upload_information[0],
-                                key=upload_information[1])
 
-    print('Grader id: %s\n' % grader_id)
+    try:
+        grader_id = register_grader(auth,
+                                    args,
+                                    bucket=upload_information[0],
+                                    key=upload_information[1])
+
+        print('Grader id: %s\n' % grader_id)
 
-    return (update_assignments(auth, grader_id, args)
-            if (args.item is not None and args.part is not None)
-            else 0)
+        return (update_assignments(auth, grader_id, args)
+                if (args.item is not None and args.part is not None)
+                else 0)
+    except:
+        print()
```

Comment:
Should we print something lie - "register grader failed"?

Reply:
That's already handled by the register_grader function!